### PR TITLE
Client Starter [minor]

### PR DESCRIPTION
**Pull request by ric-evans**
A resource script for spawning clients on a condor cluster

see https://github.com/WIPACrepo/SkyDriver/issues/5

### CODE REVIEW

**Comment by ric-evans** [comment]
fsdfsdf
ssdsd

**Comment by ric-evans** [comment]
sdfsdf

File: resources/client_starter.py [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+    LOGGER.info(job_description)
+
+    # dryrun?
+    if args.dryrun:
+        LOGGER.error("Script Aborted: Condor job not submitted")
+        return
+
+    # make connections -- do these before submitting so we don't have any unwanted surprises
+    skydriver_rc, scan_id = connect_to_skydriver()
+    schedd = get_schedd(args.collector_address, args.schedd_name)
```
Comment (dsschult) [inline resources/client_starter.py:279]: the args are called `collector` and `schedd`.  maybe they need to be renamed?

**Review by dsschult — CHANGES_REQUESTED** [review]
Generally looks good.  My only comment is that it would be nice to have a mode where the collector/schedd was filled in from local host values, so it could connect automatically for someone like Tianlu.  But it would probably work as-is.

**Comment by ric-evans** [comment]
> Generally looks good. My only comment is that it would be nice to have a mode where the collector/schedd was filled in from local host values, so it could connect automatically for someone like Tianlu. But it would probably work as-is.

I'm thinking that's what will happen when the CL args are not given (`""`)

**Review by dsschult — APPROVED** [review]

**Pull request merged**